app/services/rapid_ready_for_decision/views/hypertension_pdf.erb

```diff
@@ -0,0 +1,139 @@
+Hypertension Rapid Ready for Decision | Claim for Increase <br />
+<h1>
+  VHA Hypertension Data Summary for <br/>
```

Comment:
Do you want to start generalizing this template, e.g.:
```suggestion
  VHA <%= disability_title %> Data Summary for <br/>
```
Also rename the file to something like `rrd_data_summary_pdf.erb` or something that matches the [`VAMC_Hypertension_Rapid_Decision_Evidence.pdf` filename](https://github.com/department-of-veterans-affairs/vets-api/blob/113697926ea68cbf3c8c638cd36064c1e05a20c0/app/services/rapid_ready_for_decision/hypertension_upload_manager.rb#L15).